src/Geolocator.Plugin/Apple/GeolocatorImplementation.apple.cs

```diff
@@ -512,26 +517,36 @@ void OnLocationsUpdated(object sender, CLLocationsUpdatedEventArgs e)
 		void UpdatePosition(CLLocation location)
 		{
 			var p = (lastPosition == null) ? new Position() : new Position(this.lastPosition);
+			p.HasAccuracy = true;
 
-			if (location.HorizontalAccuracy > -1)
+			if (location.HorizontalAccuracy >= 0)
```

Comment:
Not sure why this needs to be changed?

Reply:
I just put it in there as it seemed more appropriate based on Apple docs.  They mention an error as any negative number, which technically cloud also include numbers between 0 and -1.

Ultimately, it doesn't correspond to the goal of the pull request, so I'll yank it.